release/src/main/scripts/download_github_actions_artifacts.py

```diff
@@ -0,0 +1,314 @@
+GH_API_URL_WORKFLOW_RUN_FMT = "https://api.github.com/repos/{repo_url}/actions/runs/{run_id}"
+GH_WEB_URL_WORKLOW_RUN_FMT = "https://github.com/{repo_url}/actions/runs/{run_id}"
+
+
+def parse_arguments():
+  """
+  Gets all neccessary data from the user by parsing arguments or asking for input.
+  Return: github_token, user_github_id, repo_url, release_branch, release_commit, artifacts_dir
+  """
+  parser = argparse.ArgumentParser(
+      description=
+      "Script for downloading GitHub Actions artifacts from 'Build python wheels' workflow."
+  )
+  parser.add_argument("--github-user", required=True)
+  parser.add_argument("--repo-url", required=True)
+  parser.add_argument("--release-branch", required=True)
+  parser.add_argument("--release-commit", required=True)
+  parser.add_argument("--artifacts_dir", required=True)
+
+  args = parser.parse_args()
+  github_token = ask_for_github_token()
+
+  print("You passed following arguments:")
+  pprint.pprint({**vars(args), **{"github_token": github_token}})
+
+  if not get_yes_or_no_answer("Do you want to continue?"):
+    print("You said NO. Quitting ...")
+    sys.exit(1)
+
+  user_github_id = args.github_user
+  repo_url = args.repo_url
+  release_branch = args.release_branch
+  release_commit = args.release_commit
+  artifacts_dir = args.artifacts_dir
+
+  return github_token, user_github_id, repo_url, release_branch, release_commit, artifacts_dir
+
+
+def ask_for_github_token():
+  """Ask for github token and print basic information about it."""
+  url = "https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token"
+  message = (
+      f"You need to have a github access token with public_repo scope. "
+      f"More info about creating access tokens can be found here {url}")
+  print(message)
+  github_token = input("Enter github token: ")
+  if not github_token:
+    return ask_for_github_token()
+  return github_token
+
+
+def request_url(url, github_token, return_json=True, *args, **kwargs):
+  """Helper function form making requests authorized by GitHub token."""
+  r = requests.get(url, *args, auth=("token", github_token), **kwargs)
+  if return_json:
+    r.raise_for_status()
+    return r.json()
+  return r
+
+
+def safe_get(data, key, url=None):
+  """Gets data by the key with informative Exception in case of non existent key."""
+  if key not in data:
+    message = f'There is missing key: "{key}" in response data: {data}.'
+    if url:
+      message += f" Requested url: {url}"
+    raise ValueError(message)
+  return data.get(key)
+
+
+def get_yes_or_no_answer(question):
+  """Asks yes or no question."""
+  reply = str(input(question + " 'y' or 'n'): ")).lower().strip()
+  if reply == "y":
+    return True
+  elif reply == "n":
+    return False
+  else:
+    return get_yes_or_no_answer("Uhhhh... please enter")
+
+
+def get_build_wheels_workflow_id(repo_url, github_token):
+  """Gets workflow id."""
```

Comment:
The docstring
  """Gets workflow id"""
is not adding any information since it repeats the function name. You can expand or remove the docstring.
For example: """Gets the ID of the Github Actions workflow responsible for building wheels."""

Reply:
Thank you, I will apply your suggestion.